pkgs/applications/office/planner/default.nix

```diff
@@ -10,24 +10,26 @@
 , python
 }:
 
-let version = "20170425";
+let version = "20180325";
```

Comment:
Shouldn't this be `unstable-2018-03-25`?

Reply:
Yes it should.